src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -71,6 +79,45 @@ public JwtHeader(SigningCredentials signingCredentials)
         }
 
         /// <summary>
+        /// Gets or sets the <see cref="IDictionary{TKey, TValue}"/> used to map
+        /// the JWA identifiers with the corresponding security algorithms.
+        /// </summary>
+        public static IDictionary<string, string> InboundAlgorithmMap { get; } = new Dictionary<string, string>
```

Comment:
@tushargupta51 I'm fine opting for an instance dictionary. But in this case, it will probably have to be moved to the handler itself, since the mapping is done in `JwtHeader`'s constructor.